alerts/management/commands/run_cron_alerts.py

```diff
@@ -51,7 +54,7 @@ def get_success_rate(self, monitor):
         success_rate = 100
         if success_count['total'] != 0:
             success_rate = success_count['s'] / (success_count['total']) * 100
-        return success_rate
+        return success_rate, start_time.strftime("%d %b %Y, %H:%M:%S"), end_time.strftime("%d %b %Y, %H:%M:%S")
```

Comment:
ini dibikin variabel dulu aja hug
```suggestion
        formatted_success_rate = round(float(success_rate), 2)
        formatted_start_time = start_time.strftime("%d %b %Y, %H:%M:%S")
        formatted_end_time = end_time.strftime("%d %b %Y, %H:%M:%S")
        return formatted_success_rate , formatted_start_time, formatted_end_time
```